add links to release notes, light refactor of landing page (#11051)
apache/druid: docs/design/index.md

2. **Scalable distributed system.** Typical Druid deployments span clusters ranging from tens to hundreds of servers. Druid can ingest data at the rate of millions of records per second while retaining trillions of records and maintaining query latencies ranging from the sub-second to a few seconds.
3. **Massively parallel processing.** Druid can process each query in parallel across the entire cluster.
4. **Realtime or batch ingestion.** Druid can ingest data either real-time or in batches. Ingested data is immediately available for
querying.
5. **Self-healing, self-balancing, easy to operate.** As an operator, you add servers to scale out or
remove servers to scale down. The Druid cluster re-balances itself automatically in the background without any downtime. If a
Druid server fails, the system automatically routes data around the damage until the server can be replaced. Druid
is designed to run continuously without planned downtime for any reason. This is true for configuration changes and software
updates.

> 6. **Cloud-native, fault-tolerant architecture that won't lose data.** After ingestion, Druid safely stores a copy of your data in [deep storage](architecture.md#deep-storage). Deep storage is typically cloud storage, HDFS, or a shared filesystem. You can recover your data from deep storage even in the unlikely case that all Druid servers fail. For a limited failure that affects only a few Druid servers, Druid uses replication to ensure that queries are still possible during system recovers.

Reviewer: I think "recovers" -> "recoveries", although the original seems to refer a little more clearly to "For a limited failure that affects..":

```suggestion
6. **Cloud-native, fault-tolerant architecture that won't lose data.** After ingestion, Druid safely stores a copy of your data in [deep storage](architecture.md#deep-storage). Deep storage is typically cloud storage, HDFS, or a shared filesystem. You can recover your data from deep storage even in the unlikely case that all Druid servers fail. For a limited failure that affects only a few Druid servers, replication ensures that queries are still possible during system recoveries.
```

7. **Indexes for quick filtering.** Druid uses [Roaring](https://roaringbitmap.org/) or
[CONCISE](https://arxiv.org/pdf/1004.0403) compressed bitmap indexes to create indexes to enable fast filtering and searching across multiple columns.

> 8. **Time-based partitioning.** Druid first partitions data by time. YOu can optionally implement additional partitioning based upon other fields.

Reviewer: ```suggestion
8. **Time-based partitioning.** Druid first partitions data by time. You can optionally implement additional partitioning based upon other fields.
```

Time-based queries only access the partitions that match the time range of the query which leads to significant performance improvements.
9. **Approximate algorithms.** Druid includes algorithms for approximate count-distinct, approximate ranking, and
computation of approximate histograms and quantiles. These algorithms offer bounded memory usage and are often
substantially faster than exact computations. For situations where accuracy is more important than speed, Druid also
offers exact count-distinct and exact ranking.
10. **Automatic summarization at ingest time.** Druid optionally supports data summarization at ingestion time. This

> summarization partially pre-aggregates your data potentially leading to significant cost savings and performance boosts.

Reviewer: The rewording is good, but it runs together without a comma, I think:
  
```suggestion
summarization partially pre-aggregates your data, potentially leading to significant cost savings and performance boosts.
```

## When to use Druid

Druid is used by many companies of various sizes for many different use cases. For more information see
[Powered by Apache Druid](/druid-powered).

> Druid is likely a good choice if your use case has matches a few of the following:

Reviewer: ```suggestion
Druid is likely a good choice if your use case matches a few of the following:
```
